src/python/tests/appengine/handlers/jobs_test.py

```diff
@@ -13,13 +13,15 @@
 # limitations under the License.
 """Tests for jobs."""
 import collections
+import mock
```

Comment:
Use `from unittest import mock` instead to use the one from standard libs.

Reply:
done 👍